LICENSE.md

```diff
@@ -0,0 +1,173 @@
+# Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International
```

Comment:
Was this file originally in markdown format?
Some parts are rendering weird:

![image](https://user-images.githubusercontent.com/29712657/112868919-808a6a80-90b4-11eb-97dc-5192d8826446.png)

![image](https://user-images.githubusercontent.com/29712657/112869157-bb8c9e00-90b4-11eb-9751-600432a213d8.png)

![image](https://user-images.githubusercontent.com/29712657/112869121-b4659000-90b4-11eb-9889-ba453d696232.png)

Reply:
I expect we've copied this fro mthe dev-docs https://github.com/microbit-foundation/dev-docs/blob/master/LICENSE.MD which did have the same formating. I have updated the dev-docs based on a template I found at https://github.com/idleberg/Creative-Commons-Markdown/blob/master/4.0/by-nc-sa.markdown, but I wonder if we actually want non-commercial restriction here or not? Maybe `by-sa` is more appropriate as people might use these images in commercial applications

Reply:
I've mentioned this a couple of times in the past, but never truly drove it up. We can try to change it when Jonny is back, but let's include the current license in the meantime.

Reply:
Was a copy from mark down. mark has made updates to dev doxc licence, I'll copy from that